### PR TITLE
test: add e2e test for ssl and consumer

### PR DESCRIPTION
**Pull request by nic-chen**

### CODE REVIEW

File: api/route/base_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package route
+
+import (
+	"github.com/api7/apitest"
```
Comment (juzhiyuan) [inline api/route/base_test.go:20]: @moonming  Not sure if it’s ok to include this package?

Comment (nic-chen) [reply api/route/base_test.go:20]: it's MIT License

Comment (juzhiyuan) [reply api/route/base_test.go:20]: OK

**Comment by nic-chen** [comment]
Encountered occasional problems, reopen later

**Pull request merged**